bot/cogs/doc.py

```diff
@@ -150,16 +175,37 @@ def __init__(self, bot: commands.Bot):
         """
         self.base_urls[package_name] = base_url
 
-        fetch_func = functools.partial(intersphinx.fetch_inventory, config, '', inventory_url)
-        for _, value in (await self.bot.loop.run_in_executor(None, fetch_func)).items():
-            # Each value has a bunch of information in the form
-            # `(package_name, version, relative_url, ???)`, and we only
-            # need the relative documentation URL.
-            for symbol, (_, _, relative_doc_url, _) in value.items():
-                absolute_doc_url = base_url + relative_doc_url
-                self.inventories[symbol] = absolute_doc_url
-
-        log.trace(f"Fetched inventory for {package_name}.")
+        package = await self._fetch_inventory(inventory_url, config)
+        if package:
```

Comment:
There's no reason to indent all items contained within this if. It's better to invert the if and dedent the contents:
```py
        package = await self._fetch_inventory(inventory_url, config)
        if not package:
            log.trace(f"No inventory for {package_name}.")
            return

        for group, value in package.items():
```